src/modules/installers/shared/actions.ts

```diff
@@ -91,28 +91,42 @@ export async function getPackageJson(
 // Helper to add dependencies to a package.json (without installing)
 export async function addDependenciesFromPackage(
   pkg: PackageJson,
-): Promise<void> {
+): Promise<{ labelaDependencies: string[] | null }> {
   const { projectName } = state.answers;
 
-  const dependencies = pkg.dependencies
-    ? Object.keys(pkg.dependencies).join(' ')
-    : null;
+  const dependencies = pkg.dependencies ? Object.keys(pkg.dependencies) : null;
+
+  // Filter out dependencies that are LabelA internal
+  const npmDependencies = dependencies?.filter(
+    (d) => !d.includes('@labela/components'),
+  );
+  const labelaDependencies = dependencies?.filter((d) =>
+    d.includes('@labela/components'),
+  );
 
-  if (dependencies) {
+  if (npmDependencies && npmDependencies.length > 0) {
     await asyncExec(
-      `npx add-dependencies ${projectName}/package.json ${dependencies}`,
+      `npx add-dependencies ${projectName}/package.json ${npmDependencies.join(
+        ' ',
+      )}`,
     );
   }
 
   const devDependencies = pkg.devDependencies
-    ? Object.keys(pkg.devDependencies).join(' ')
+    ? Object.keys(pkg.devDependencies)
     : null;
 
-  if (devDependencies) {
+  if (devDependencies && devDependencies.length > 0) {
     await asyncExec(
-      `npx add-dependencies ${projectName}/package.json ${devDependencies} -D`,
+      `npx add-dependencies ${projectName}/package.json ${devDependencies.join(
+        ' ',
+      )} -D`,
     );
   }
+
+  return {
+    labelaDependencies: labelaDependencies || null,
```

Comment:
Ik zou dit in een andere function doen. `addDependenciesFromPackage` dat alleen `labelaDependencies` returned heeft niet veel met elkaar te maken.

Reply:
Heb je gelijk in. Heb het aangepast in deze commit, laat maar weten wat je vind: https://github.com/react-prime/create-react-prime/pull/21/commits/2c6cd4d954c82ea88db317b904bb9f50b88b2a26